pkg/services/accesscontrol/resourcepermissions/service.go

```diff
@@ -79,6 +80,14 @@ func New(options Options, router routing.RouteRegister, ac accesscontrol.AccessC
 	return s, nil
 }
 
+// PermissionService interface allows mocking Service that is defined below
+type PermissionService interface {
```

Comment:
We already have a interface in the access control package, see https://github.com/grafana/grafana/blob/main/pkg/services/accesscontrol/accesscontrol.go#L36